runtime/vm/JFRChunkWriter.hpp

```diff
@@ -185,6 +186,7 @@ class VM_JFRChunkWriter {
 	static constexpr int THREAD_CONTEXT_SWITCH_RATE_SIZE = sizeof(float) + (3 * sizeof(I_64));
 	static constexpr int THREAD_STATISTICS_EVENT_SIZE = (6 * sizeof(U_64)) + sizeof(U_32);
 	static constexpr int THREAD_DUMP_EVENT_SIZE_PER_THREAD = 1000;
+	static constexpr int NATIVE_LIBRARY_ADDRESS_SIZE = sizeof(I_64) + (2 * sizeof(UDATA)) + (2 * sizeof(U_8));
```

Comment:
None of these size estimates are strictly correct (if a 64-bit value must be written, they add 8 (`sizeof(U_64)`) to the total when, pessimistically, they should assume 9 bytes of space will be required). For now, please follow the (broken) pattern and assume 8 bytes for each 64-bit field. There are seven fields (size, type, ticks, name-encoding, name-length, address low, address high); four are 64-bit fields, two are `UDATA`, and one (name-encoding) can be assumed to be shorter. Thus, this should be:
```c
	static constexpr int NATIVE_LIBRARY_ADDRESS_SIZE = (4 * sizeof(U_64)) + (2 * sizeof(UDATA)) + sizeof(U_8);
```